### ISSUE TITLE
🐛 BUG: wrangler secret:bulk (still) returns `exit code 0` on failure

### ISSUE THREAD

**Issue by aroman** [cloudflare/workers-sdk#3728]
### Which Cloudflare product(s) does this pertain to?

Wrangler core

### What version(s) of the tool(s) are you using?

3.5.0

### What operating system are you using?

any

### Describe the Bug

what it says on the tin. repro:

```
bash-3.2$ echo 'INVALID JSON' | npx wrangler secret:bulk
 ⛅️ wrangler 3.5.0
------------------
🌀 Creating the secrets for the Worker "magiccircle-gg"
✘ [ERROR] 🚨 Please provide a JSON file or valid JSON pipe


bash-3.2$ echo $?
0
bash-3.2$ npx wrangler --version
 ⛅️ wrangler 3.5.0
------------------
```

per #3704 [landing in 3.5.0](https://github.com/cloudflare/workers-sdk/releases/tag/wrangler%403.5.0), I would expect a non-zero return code.

cc @JacobMGEvans

**Comment by JacobMGEvans** [comment]
Thanks for the report, I will look into getting it returning the correct exit code on failure.

**Comment by RamIdeas** [comment]
Hi @aroman! Can you try on the latest version and let us know if this issue is still present please?

**Comment by aroman** [comment]
Hi @RamIdeas, unfortunately, _all_ of secret uploading for my workflow due to a separate regression introduced in v3.17: https://github.com/cloudflare/workers-sdk/issues/4496

So, until that issue is resolved, I am unable to test this one.

**Comment by RamIdeas** [comment]
Hi @aroman! Thanks for noting the dependency! I saw that issue was closed as completed yesterday and released yesterday in wrangler@3.23.0 – can you try on that version and let us know if this issue is still present please?

**Comment by lrapoport-cf** [comment]
hey @aroman :) sounds like the issue here is resolved and we're prioritizing https://github.com/cloudflare/workers-sdk/issues/4562, so will close this issue out. thanks!
